lib/faktory_worker/worker.ex

```diff
@@ -186,7 +186,7 @@ defmodule FaktoryWorker.Worker do
     # set a timeout for the job process of the configured reserve_for
     # time minus 20 seconds to ensure the job is stopped before faktory
     # can expire and retry it on the server
-    Process.send_after(self(), :job_timeout, reserve_for - 20)
+    Process.send_after(self(), :job_timeout, (reserve_for - 20) * 1000)
```

Comment:
I took the opportunity to correct this since `send_after` takes milliseconds and the reserve for is in seconds. 

Currently we are making the timeout 1000x shorter 😱